### PR TITLE
Task 8 - add comment

### PR DESCRIPTION
**Pull request by OliverHolt**
add ability to add comment to an article

### CODE REVIEW

File: src/components/AddComment.jsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    postComment(newComment, article_id, username)
+      .then((newComment) => {
+        setNewComment("");
+        setUsername("");
+        setArticleComments((currComments) => {
+          const newCommentList = [...currComments];
+          newCommentList.unshift(newComment);
+          return newCommentList;
+        });
+        alert("Comment added!");
```
Comment (callumu21) [inline src/components/AddComment.jsx:20]: Stay away from alerts, as you wouldn't really expect these on a real site. Could you turn this into a text display under the comment box that displays the validation or error message depending on whether the request was successful?

File: src/components/AddComment.jsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        <option value="cooljmessy">cooljmessy</option>
+        <option value="weegembump">weegembump</option>
+        <option value="jessjelly">jessjelly</option>
+      </select>
+      <br />
+      <br />
+      <button
+        id="add"
+        onClick={() => {
+          setAddButton(true);
```
Comment (callumu21) [inline src/components/AddComment.jsx:68]: Do you ever reset this value to false after the comment has been added?

**Review by callumu21 — CHANGES_REQUESTED** [review]
Great start, Oliver, and good mobile-first design. However, your user messages shouldn't be alerts, so think about how you could change them as per my comment. Also, users should be logged in to comment, rather than just choosing a username in the dropdown. I've put two ways of handling this in a comment, but let me know if you have any questions

**Comment by JVal-NC** [comment]
All ok, go ahead and merge

**Pull request merged**